src/lib/mongoBackend/TriggeredSubscription.cpp

```diff
@@ -97,9 +99,14 @@ TriggeredSubscription::~TriggeredSubscription()
 * TriggeredSubscription class is shared for NGSI9 and NGSI10 subscriptions, so it is better
 * to keep expressions (an artifact for NGSI10) out of the constructor, in its independent fill
 * method
-*
 */
-void TriggeredSubscription::fillExpression (const std::string& q, const std::string& georel, const std::string& geometry, const std::string& coords)
+void TriggeredSubscription::fillExpression
+(
+  const std::string&  q,
+  const std::string&  georel,
+  const std::string&  geometry,
+  const std::string&  coords
+)
 {
```

Comment:
I changed this in the "mem leak branch", not to use a pointer but a StringFilter object.
If moved here I will do the same.
TriggeredSubscription will have a StringFilter pointer, pointing to the sub.cache and a StringFilter object, used when the cache is not used.

Actually, I got the impression that lots of info is stored in both cache and in TriggeredSubscription. The whole mechanish should be looked over.

TriggeredSubscription is older than subCache and should probably have changed when the subCache was implemented.